solo-io/gloo
# [DNM] Validate GHA Cache Miss

# Description
Validate that a change with some go.mod updates, results in a github action cache miss

**This is not meant to merge, just demonstrate behavior, and then be closed**

# Context

In https://github.com/solo-io/gloo/pull/8545 we proved that cache hits work. This PR demonstrates that cache misses occur as well.

<img width="1064" alt="Screen Shot 2023-08-07 at 12 26 21 PM" src="https://github.com/solo-io/gloo/assets/5279627/6bcd5636-9943-4dca-8a45-6db166c76589">

## Testing steps

- Confirm that codegen step experienced a cache miss
- Confirm that GHA 1.14.x cache is not updated 
- Confirm that a new cache entry exists for this PR

## Notes for reviewers
-

# Checklist:
